rfcs/protocols/asset-transfer/corda.md

```diff
@@ -6,9 +6,19 @@
 # Asset Transfer Protocol Units in Corda Networks
 
 - RFC: 02-010
-- Authors: Venkatraman Ramakrishna, Sandeep Nishad, Krishnasuri Narayanam, Dhinakaran Vinayagamurthy
+- Authors: Krishnasuri Narayanam, Sandeep Nishad, Venkatraman Ramakrishna, Dhinakaran Vinayagamurthy
 - Status: Proposed
-- Since: 01-Jun-2021
+- Since: 21-Oct-2022
 
-## Summary
+## Weaver Asset Transfer implementation approach for Corda network
+Below we describe the approach for the implementation of asset transfer from one ledger to another that requires only non-validating notary.
+- Pledge transaction: Consumes an input *Asset State* and produces *Pledge State* as the output. A reference to the consumed *Asset State* is maintained as a Corda *StaticPointer* (instead of copying the entire *Asset State* object), which will be used during the reclaim of pledged asset later in the same ledger.
```

Comment:
Are "StaticPointer" and "Asset State" keywords? Can you link to where they are defined? If not defined elsewhere, define them in this file.

Reply:
Done